.buildkite/scripts/e2e-pipeline/generate-steps.py

```diff
@@ -38,7 +38,7 @@ def generate_steps_for_scheduler(versions) -> list:
     steps: list = []
     snapshots = versions["snapshots"]
     for snapshot_version in snapshots:
-        if snapshots[snapshot_version].startswith("7."):
+        if snapshots[snapshot_version] is None or snapshots[snapshot_version].startswith("7."):
```

Comment:
```suggestion
        if snapshots.get(snapshot_version,"").startswith("7."):
```
Nit, but this looks like the use case `get` is meant to solve.

Reply:
Ah right, makes sense!